.github/workflows/build_and_deploy.yml

```diff
@@ -78,3 +80,36 @@ jobs:
             | Additional Payments | <${{ env.APP_URL }}/additional-payments/claim> |
             | Student Loans       | <${{ env.APP_URL }}/student-loans/claim>       |
             | Admin               | <${{ env.APP_URL }}/admin>                     |
+
+  deploy_test:
+    name: Deploy to test environment
+    concurrency: deploy_test
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master' && github.event_name == 'push'
+    needs: [build]
+    environment:
+      name: test-aks
+      url: ${{ steps.deploy.outputs.environment_url }}
+    outputs:
+      environment_url: ${{ steps.deploy.outputs.environment_url }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - uses: azure/login@v2
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - uses: ./.github/actions/deploy-environment
+        id: deploy
+        with:
+          environment: test-aks
+          docker-image: ${{ needs.build.outputs.docker-image-tag }}
+          azure-credentials: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: Run migrations
+        shell: bash
+        run: |
+          make ci test-aks get-cluster-credentials
+          kubectl exec -n srtl-test deployment/claim-additional-payments-for-teaching-test-worker -- sh -c "DISABLE_DATABASE_ENVIRONMENT_CHECK=1 bin/prepare-database"
```

Comment:
We normally have a smoke test post deployment for permanent env deployments (i.e. test/prod), and also look to have a slack msg generated on workflow failure (of perm env deploy/smoke test failure)

Reply:
https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/2875

Reply:
slack message PR: https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/2864